### PR TITLE
1.17 - Last-minute string cleanups

### PR DESCRIPTION
**Pull request by doofus-01**
This should make the fluff tied to the ravens less placeholder-y.  I'll see if there are any other strings I added that need a quick revision before the freeze, but if 1.17.22 can't wait any longer, this can be merged (assuming it doesn't fail CI for some reason).

EDIT: As much as I dislike the current name of the jumping spiders, I'm having trouble thinking of a clear improvement.
EDIT2:  This is probably all I'll get done, will merge in a couple hours

### CODE REVIEW

**Comment by stevecotton** [comment]
Several instances of double-spaces after a full stop.

File: data/core/units.cfg [hunk 1]
```diff
@@ -355,7 +355,9 @@ Orcs who were not the strongest of their litter tend to specialize in other skil
...
 
-No predators have the numbers to rival orcs or humans, so these races have constant interaction with ravens.  In the aftermath of a battle, the dead and wounded are a banquet for ravens, thus ravens are a nuisance for allies of the defeated but are seen by some victors as part of martial ceremonies.  As intelligent scavengers, ravens do not need war dead to survive.  Some farmers struggle to keep ravens away from freshly sown seeds, but others are happy to have them around as sentries."
+In the aftermath of a battle, the dead and wounded are a banquet for ravens, thus ravens are a nuisance for allies of the defeated but are seen by some victors as part of martial ceremonies.  The violence and large numbers that characterize both orcs and humans bring these races into constant interaction with ravens in this context.  For this reason, the birds have become characters in fables, parables, and superstitions related to war.
```
Comment (stevecotton) [inline data/core/units.cfg:358]: "large numbers" seems more like it should be "large numbers of casualties". However, it feels like too much dwelling on the consequences of violence, ignoring the normal handwave of video games forgetting about the victims.

File: data/core/units.cfg [hunk 2]
```diff
@@ -355,7 +355,9 @@ Orcs who were not the strongest of their litter tend to specialize in other skil
...
+
+As intelligent scavengers, ravens do not need war dead to survive.  Some farmers struggle to keep ravens away from freshly sown seeds, but others are happy to have them around as sentries."
```
Comment (stevecotton) [inline data/core/units.cfg:360]: Sentries against what?

Comment (doofus-01) [reply data/core/units.cfg:360]: Intruders, visitors, anything.

File: data/core/units.cfg (hunk 1 above)
Comment (doofus-01) [reply data/core/units.cfg:358]: Large numbers could also be the populations in general.  Either interpretation seems OK to me.

**Comment by doofus-01** [comment]
> Several instances of double-spaces after a full stop. They're already in the text, so this isn't a regression, but it would still be good to fix before the string freeze.

Maybe a bit of a tangent, but what are the chances we can ever drop that rule?  Does double-space break anything?

**Comment by Pentarctagon** [comment]
At this point I think it's not that it'd break anything so much as it is for consistency with all the rest of the game's text.

**Comment by doofus-01** [comment]
OK.  It will be an ongoing issue, since double-space was pounded into me as a student and I've never heard a convincing argument to break the habit (and make monospace harder to read).  But I'll try here.

**Pull request merged**